src/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2020-2021 Fabien Potencier
```

Comment:
This one should be at the root, not in `src/`

Reply:
thanks, fixed in 57e02174f961aa537b58428ad7edecf12d879a5e